File: Selenium/Activities/Activity15.java
package com.example.fst_m1_selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity15 {
	
public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://v1.training-support.net/selenium/dynamic-attributes");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		driver.findElement(By.cssSelector("input[class^='username-']")).sendKeys("admin");
		
		driver.findElement(By.xpath("//input[starts-with(@class,'password-')]")).sendKeys("password");
		
		driver.findElement(By.xpath("//button[text()='Log in']")).click();
		
		String message = driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText();
		
		System.out.println("Login message: "+ message);
		
		driver.quit();	
		
	}

}

File: Selenium/Activities/Activity15.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-attributes")

    print("Title of the page: ",driver.title)

    driver.find_element(By.CSS_SELECTOR, "input[class^='username-']").send_keys("admin")

    driver.find_element(By.XPATH, "//input[starts-with(@class,'password-')]").send_keys("password")

    driver.find_element(By.XPATH, "//button[text()='Log in']").click()

    message = driver.find_element(By.XPATH, "//div[@id='action-confirmation']").text

    print("Login message: ",message)

    driver.quit()